pokemongo_bot/__init__.py

```diff
@@ -343,6 +359,120 @@ def drop_item(self, item_id, count):
         #{'responses': {'RECYCLE_INVENTORY_ITEM': {'result': 1, 'new_count': 46}}, 'status_code': 1, 'auth_ticket': {'expire_timestamp_ms': 1469306228058L, 'start': '/HycFyfrT4t2yB2Ij+yoi+on778aymMgxY6RQgvrGAfQlNzRuIjpcnDd5dAxmfoTqDQrbz1m2dGqAIhJ+eFapg==', 'end': 'f5NOZ95a843tgzprJo4W7Q=='}, 'request_id': 8145806132888207460L}
         return inventory_req
 
+    def check_inventory_space(self, items_count, max_storage, items):
```

Comment:
Can you put this in a new worker called like DropItemsWorker or something? This is a lot of code to be adding to __init__